ssdeep/ssdeep-lib/uninstall-ssdeep.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+TMP_PATH=/var/tmp/
+MAKE=make
+SUDO=sudo
+REP_PATH=`pwd`
+
+cd ${TMP_PATH}
+wget -O ssdeep.tar.gz https://sourceforge.net/projects/ssdeep/files/ssdeep-2.13/ssdeep-2.13.tar.gz/download
+tar zxvf ssdeep.tar.gz
+cd ssdeep-2.13/
+cp ${REP_PATH}/ssdeep/ssdeep-lib/ac-fix/* ./
+./configure
```

Comment:
If one would want to install ssdeep-lib to a custom path, he can do it manually.